numpy/lib/tests/test_recfunctions.py

```diff
@@ -214,6 +214,8 @@ def test_structured_to_unstructured(self):
                      dtype=[('x', 'i4'), ('y', 'f4'), ('z', 'f8')])
         out = np.mean(structured_to_unstructured(b[['x', 'z']]), axis=-1)
         assert_equal(out, np.array([ 3. ,  5.5,  9. , 11. ]))
+        out = np.mean(structured_to_unstructured(b[['x']]), axis=-1)
+        assert_equal(out, np.array([ 1. ,  4. ,  7. , 10. ]))
```

Comment:
Previously returned the scalar `5.5`

Reply:
If someone actually wants the result `5.5`, they should just have been using `np.mean(b['x'], axis=-1)`